evennia/contrib/rpg/rpsystem/rpsystem.py

```diff
@@ -1539,27 +1397,26 @@ def get_display_name(self, looker, **kwargs):
 
         Keyword Args:
             pose (bool): Include the pose (if available) in the return.
+            ref (str):   Specifies the capitalization for the displayed name.
 
         Returns:
             name (str): A string of the sdesc containing the name of the object,
             if this is defined.
                 including the DBREF if this user is privileged to control
                 said object.
 
-        Notes:
-            The RPObject version doesn't add color to its display.
-
         """
-        idstr = "(#%s)" % self.id if self.access(looker, access_type="control") else ""
+        idstr = "(#%s)" % self.id if self.access(looker, access_type="control") and not kwargs.get("no_id", False) else ""
+        ref = kwargs.get("ref","~")
```

Comment:
It would be an idea to also update the `%s` when you are at it (the original code is old enough to predate things like .format and f-strings)